final/EU/FIN/EFRO_21_readme.txt

```diff
@@ -0,0 +1,37 @@
+Endless ATC Rovaniemi v1.0 by Zac Kaplowitz

```

Comment:
Consider renaming to .md and adding some basic [markdown](https://guides.github.com/features/mastering-markdown/) formatting?